File: voitseshchuk/papka lab№2/port.py
class Port:
    def __init__(self, ID, latitude, longitude):
        self.ID = ID
        self.coordinates = (latitude, longitude)
        self.storage = []

    def load_container(self, container):
        # Додати логіку для завантаження контейнера в порт
        pass

    def unload_container(self, container):
        # Додати логіку для розвантаження контейнера з порту
        pass

    def find_closest_refueling_port(self):
        # Додати логіку для пошуку найближчого порту для підйому палива
        pass

File: dmytryshyn/lab4/Lab4DataLayer/Entities/Port.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab4DataLayer.Entities
{
    public class Port
    {
        public int Id { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }

        public virtual ICollection<Container> Containers { get; set; }
        public virtual ICollection<Ship> Ships { get; set; }


    }
}
